view/templates/filebrowser.tpl

```diff
@@ -40,7 +40,7 @@
 	</div>
 
 	<div class="upload">
-		<button id="upload-{{$type}}"><img id="profile-rotator" src="images/rotator.gif" alt="{{$wait}}" title="{{$wait|escape:'html'}}" style="display: none;" /> {{"Upload"|t}}</button>
+		<button id="upload-{{$type}}"><img id="profile-rotator" src="images/rotator.gif" alt="{{$wait}}" title="{{$wait|escape:'html'}}" style="display: none;" /> {{$upload}}</button>
```

Comment:
the new translation marking cannot be used within the templates anymore? Is it because it's now in a class?

Reply:
Apparently it is possible but @zeroadam went instead for the way we have been using so far by avoiding function calls in the template.